src/Rules.py

```diff
@@ -102,14 +102,14 @@ def checkRequireStringForArea(state: CollectionState, area: dict):
             item_base = item
             item = item.lstrip('|@$').rstrip('|')
 
-            item_parts = item.split(":")
+            item_parts = item.split(":")  # type: list[str]
             item_name = item
```

Comment:
Personally I would add a strip() around here for when there's only 1 part, but thats not 100% required soo ill leave it to your opinion,
other then that looks good